src/pages/CreateForm/CreateForm.js

```diff
@@ -77,25 +79,28 @@ class CreateForm extends React.Component {
             } else if(this.state.redirect) {
                 return <Navigate to="/records"/>
             } else {
-                return (
-                    <div className="page-container">
-                        <p className="form-creation-title">Form Creation</p>
-                        <div className="form-options">
-                            <div className="form-option">
-                                <p>What program is this form associated with?</p>
-                                <input name="program" onChange={this.handleUpdate} type="text"/>
-                            </div>
-                            <div className="form-option">
-                                <p>Content</p>
-                                <textarea name="content" onChange={this.handleUpdate}/>
-                            </div>
+            return (
+                <div className="page-container" id="form-create-page">
+                    <div id="form-create-header">
+                        <img src={Logo} alt="MissionSAFE logo" />
+                        <p className="form-creation-title">Create Form</p>
+                    </div>
+                    <div className="form-option">
+                        <div className="form-option">
+                            <p className="form-label">What program is this form associated with?</p>
+                            <input name="program" onChange={this.handleUpdate} type="text" />
+                        </div>
```

Comment:
When creating a form, we also need "name" and "description" fields. Attached the image below for reference.
<img width="617" alt="Screenshot 2023-04-16 at 10 33 58 PM" src="https://user-images.githubusercontent.com/63796975/232371796-072d1a0b-88b7-49f9-8fff-4d90df71fa1a.png">